classes/class-ledyer-ajax.php

```diff
@@ -60,8 +60,11 @@ public static function lco_wc_update_cart() {
 
 		$values = array();
 		if ( isset( $_POST['checkout'] ) ) {
-			parse_str( wp_unslash( $_POST['checkout'] ), $values );// phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
+			$response = wp_unslash( $_POST['checkout'] );
+			parse_str( html_entity_decode($response), $values );
+			$values=filter_var_array($values, FILTER_SANITIZE_ENCODED);
```

Comment:
Previously we instantly converted the received string array of url query params to an array.

Updated to first make the request, stored the response, which is the url query parameter string as `$response`.
Then use the built in `html_entity_decode` function to remove any illegal inputs. After that, parse the string to an array.

Finally filter the array to sanitize is.

Reply:
Gotta love the fact that it was consciously ignored before ❤️ 
`// phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized`